pages/documentation/settings.md

```diff
@@ -77,6 +77,12 @@ This will generate the same output as configuring the setting with default value
 );
 ```
 
+{:.site-note.radius-bottom-0}
+**Note:** We use dot notation to describe settings that are nested inside a Sass map. For example, for the code sample above, `$background-color-settings.responsive` would refer to the `responsive` property inside `$background-color-settings`.
```

Comment:
I think we should describe the environment in which we use the dot notation as well as potentially link out to a resource that tells them more about it. So then it would be like: 

```suggestion
**Note:** We use [dot notation](https://builtin.com/data-science/dot-notation) in most of our documentation to describe settings that are nested inside a Sass map. For example, for the code sample above, `$background-color-settings.responsive` would refer to the `responsive` property inside `$background-color-settings`.
```

Reply:
I like this suggestion! Wondering if a simpler link might be more appropriate for this since users don't really need to understand any of the whys or hows for using it - they just  need to know what it is and that we use it in our docs. A possible substitute: (Chose MDN because we use it as a source elsewhere on the site)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors

Happy to defer to your expertise.

Reply:
Thanks @amyleadem! I guess my only concern is that they don't know that they don't need to know the hows and whys. The MDN link provides a circular definition (using a word to define itself). Is there another source we regularly reference?

Reply:
Note:
After an offline discussion with @bonnieAcameron, I have added [this link ](https://idratherbewriting.com/learnapidoc/docapis_diving_into_dot_notation.html#dot-notation) as a resource in case users want to learn more about what dot notation is. 

I looked at MDN and other more expected resources, but could not find a simple definition. 
:warning: @thisisdano, is it ok to use a resource like this?

@bonnieAcameron I made a couple additional copy tweaks in [a6cf618](https://github.com/uswds/uswds-site/pull/2122/commits/a6cf61863c9e6539ed8e83fc87a53c1196f040ee) and [cfdaaf4](https://github.com/uswds/uswds-site/pull/2122/commits/cfdaaf41adeeb90a12f75202ca1866cc8b12bbca). Will you take another look and let me know if you need any edits?